### PR TITLE
feat: LSDV-5297: Sort phrases

### PR DESCRIPTION
**Pull request by Travis1282**
### PR fulfills these requirements
- [X] Commit message(s) and PR title follows the format `[fix|feat|ci|chore|doc]: TICKET-ID: Short description of change made` ex. `fix: DEV-XXXX: Removed inconsistent code usage causing intermittent errors`
- [X] Tests for the changes have been added/updated (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [X] Best efforts were made to ensure docs/code are concise and coherent (checked for spelling/grammatical errors, commented out code, debug logs etc.)
- [X] Self-reviewed and ran all changes on a local instance (for bug fixes/features)

#### Change has impacts in these area(s)
_(check all that apply)_
- [ ] Product design
- [X] Frontend

### Describe the reason for change
Phrases need to be sorted

#### What does this fix?
This sorts by start time 

#### What is the new behavior?
sorted by start time and null values at the end

#### What is the current behavior?
no sorting

#### What libraries were added/updated?
none

#### Does this change affect performance?
no

#### Does this change affect security?
no

#### What alternative approaches were there?
n/a

#### What feature flags were used to cover this change?
fflag_feat_front_lsdv_e_278_contextual_scrolling_short

### Does this PR introduce a breaking change?
_(check only one)_
- [ ] Yes, and covered entirely by feature flag(s)
- [ ] Yes, and covered partially by feature flag(s)
- [X] No
- [ ] Not sure (briefly explain the situation below)

### What level of testing was included in the change?
_(check all that apply)_
- [ ] e2e
- [ ] integration
- [X] unit

### Which logical domain(s) does this change affect?
_(for bug fixes/features, be as precise as possible. ex. Authentication, Annotation History, Review Stream etc.)_

### CODE REVIEW

File: src/tags/object/Paragraphs/Phrases.js [hunk 1]
```diff
@@ -4,13 +4,19 @@ import { Button } from 'antd';
...
+};
+
+export const Phrases = observer(({ item, contextScroll }) => {
   const cls = item.layoutClasses;
   const withAudio = !!item.audio;
 
   if (!item._value) return null;
-
-  const val = item._value.map((v, idx) => {
+  const list = contextScroll ? item._value.sort(sortPhrases) : item._value;
```
Comment (bmartel) [inline src/tags/object/Paragraphs/Phrases.js:18]: We will need this to be sorted within the model level so anywhere that references this data it is the same.

**Review by bmartel — CHANGES_REQUESTED** [review]

File: src/tags/object/Paragraphs/Phrases.js [hunk 2]
```diff
@@ -9,8 +9,7 @@ export const Phrases = observer(({ item }) => {
   const withAudio = !!item.audio;
 
   if (!item._value) return null;
-
-  const val = item._value.map((v, idx) => {
+  const val =  item._value.map((v, idx) => {
```
Comment (yyassi-heartex) [inline src/tags/object/Paragraphs/Phrases.js:12]: minor cleanup
```suggestion
  const val = item._value.map((v, idx) => {
```

**Review by bmartel — APPROVED** [review]
Approving based on the cleanup of the minor observations.

**Pull request merged**